### ISSUE TITLE
Unable to retrieve field rules via GraphQL

### ISSUE THREAD

**Issue by nzimelimited** [solspace/craft3-freeform#1044]
**Describe the bug or issue you're experiencing**
The field `rules` returns null for all types when using GraphQL

**Steps to reproduce**
Example query:
```
query($handle: [String]) {
  freeform {
    form(handle: $handle) {
      name
      handle
      pages {
        label
        rows {
          fields {
            id
            type
            label
            handle
            required
            rules
          }
        }
      }
    }
  }
}
```

**Expected behavior**
Rules should return JSON as per the documentation and v4 of the plugin.

**Craft & Plugin Info:**

- Craft Version: 4.7.1
- Freeform Version: 5.0.10
- Freeform Edition: Pro
- Fresh Install or Upgrade: Fresh

**Additional context**
Found this exploring the plugin a bit in `packages/plugin/src/Bundles/GraphQL/Interfaces/FieldInterface.php`
```
'rules' => [
    'name' => 'rules',
    'type' => Type::string(),
    'description' => "Field's rules",
    'resolve' => function ($source) {
         // FIXME
         return null; // $source->getRules();
    },
]
```
Guessing this might be the reason! Any ideas as to when this might be fixed?

Thanks.

**Comment by kjmartens** [comment]
Sorry about this @nzimelimited.

Conditional Rules have changed in Freeform 5. Instead of being present on each field, they are handled as JSON at the end of the form and look for `data-field-container="myFieldHandle"` on each field's container.

For GraphQL, this is a bit of an oversight for not including this. We will update this and release that early next week. 🙂

**Comment by kjmartens** [comment]
Hi, @nzimelimited,

This is now addressed in Freeform 5.0.12+. You can check out the documentation here: 🙂 
https://docs.solspace.com/craft/freeform/v5/headless/graphql/#rules-interface
